service-discovery/consul/src/test/java/io/smallrye/stork/servicediscovery/consul/ConsulServiceDiscoveryIT.java

```diff
@@ -133,6 +133,33 @@ void shouldRefetchWhenRefreshPeriodReached() throws InterruptedException {
         assertThat(instances.get().get(0).getPort()).isEqualTo(8506);
     }
 
+    @Test
+    void shouldDiscoverServiceWithSpecificName() throws InterruptedException {
+        //Given a service `my-service` registered in consul and a refresh-period of 5 seconds
+        String serviceName = "my-service";
+        TestConfigProvider.addServiceConfig("my-service", null, "consul",
+                null, Map.of("consul-host", "localhost", "consul-port", String.valueOf(consulPort), "refresh-period", "5",
+                        "application", "my-consul-service"));
+        stork = StorkTestUtils.getNewStorkInstance();
+        //Given a service `my-service` registered in consul
+        setUpServices("my-consul-service", 8406, "example.com");
```

Comment:
The same as for kube, I think it would be good if there was another service with a different name to make sure that we only select the desired one